.circleci/config.yml

```diff
@@ -0,0 +1,178 @@
+# CircleCI 2.0 configuration file
+version: 2
+
+# Re-usable blocks to reduce boilerplate in job definitions.
+references:
+  host_environment_defaults: &host_environment_defaults
+    # Customize the JVM maximum heap limit
+    JAVA_OPTS: -Xmx3200m
+  docker_environment_defaults: &docker_environment_defaults
+    # Customize the JVM to read correct memory values
+    JAVA_OPTS: '-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:MaxRAMFraction=1'
+
+  default_java_job: &default_java_job
+    docker:
+      - image: circleci/openjdk:8-jdk
+        environment:
+          <<: *docker_environment_defaults
+    environment:
+      <<: *host_environment_defaults
+    working_directory: ~/workdir
+
+  restore_source_code: &restore_source_code
+    restore_cache:
+      keys:
+        - source-code-0.0.3-{{ .Branch }}-{{ .Revision }}
+        - source-code-0.0.3-{{ .Branch }}
+        - source-code-0.0.3
+  restore_dependencies: &restore_dependencies
+    restore_cache:
+      keys:
+        - sbt-ivy2-dependencies-0.0.3-{{ checksum "/tmp/dependencies.cache.tmp" }}
+        - sbt-ivy2-dependencies-0.0.3
+  restore_build: &restore_build
+    restore_cache:
+      keys:
+        - sbt-build-0.0.3-{{ .Branch }}-{{ .Revision }}
+        - sbt-build-0.0.3
+
+  setup_dependencies_key: &setup_dependencies_key
+    run:
+      name: Generate cache key
+      command: |
+        shasum build.sbt \
+          project/plugins.sbt \
+          project/build.properties \
+          project/Common.scala \
+          project/Dependencies.scala > /tmp/dependencies.cache.tmp
+
+jobs:
+  checkout_code:
+    <<: *default_java_job
+    steps:
+      - *restore_source_code
+      - checkout
+      - save_cache:
+          key: source-code-0.0.3-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - ~/workdir
+
+  sbt_dependencies:
+    <<: *default_java_job
+    steps:
+      - *restore_source_code
+      - *setup_dependencies_key
+      - *restore_dependencies
+      - run:
+          name: Resolve dependencies
+          command: |
+            sbt update
+      - save_cache:
+          paths:
+            - ~/.ivy2
+            - ~/.sbt
+          key: sbt-ivy2-dependencies-0.0.3-{{ checksum "/tmp/dependencies.cache.tmp" }}
+
+  compile:
+    <<: *default_java_job
+    steps:
+      - *restore_source_code
+      - *setup_dependencies_key
+      - *restore_dependencies
+      - *restore_build
+      - run:
+          name: Compile
+          command: ./scripts/compile.sh
+      - save_cache:
+          paths:
+            - ~/workdir/target
+            - ~/workdir/project/target
+          key: sbt-build-0.0.3-{{ .Branch }}-{{ .Revision }}
+
+  test:
+    machine: true
+    environment:
+      <<: *host_environment_defaults
+    working_directory: ~/workdir
+    steps:
+      - *restore_source_code
+      - *setup_dependencies_key
+      - *restore_dependencies
+      - *restore_build
+      - run:
```

Comment:
After you updated this file seems like you removed the cloc installation part and the tests are failing.